CONTRIBUTING.md

```diff
@@ -10,7 +10,21 @@ patches and features.
 
 ## Development environment
 
-Next steps should work on clear Ubuntu 18.04.
+## Mac OS 10.15
+
+-   Install necessary dependencies:
+    ```sh
+    brew install git python pyenv redis curl openssl
+    ```
+
+-   Install Python 3.6
+    ```sh
+    pyenv update
+    pyenv install -v 3.6
+    pyenv global 3.6
```

Comment:
@vnishukov, python 3.8.2 is used in `Dockerfile`. Is there any special purpose for using python 3.6?

Reply:
The only reason is that it didn't work for me on MacOS Catalina 10.15.6 using Python 3.8. But it do right after downgrade Python version to 3.6.9

Reply:
@vnishukov , could you please quickly check that if you reinstall python 3.8 on mac it still doesn't work?

Reply:
I will

Reply:
Seems it still works.